package.js

```diff
@@ -1,7 +1,7 @@
 Package.describe({
   name: 'ephemer:reactive-datatables',
   summary: "Fast and reactive jQuery DataTables using standard Cursors / DataTables API. Supports Bootstrap 3.",
-  version: "1.1.0",
+  version: "1.1.1",
```

Comment:
I'd like to put the version to 1.2.0 because this change adds a major new feature and may also work differently (pagination etc.), is that alright with you?